### PR TITLE
TMA-45: add test for MdIdToUri mixin

### PR DESCRIPTION
**Pull request by hacxman**

### CODE REVIEW

File: spec/integration/mixins/id_to_uri_spec.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# LICENSE file in the root directory of this source tree.
+
+require 'gooddata/mixins/md_id_to_uri'
+
+describe GoodData::Mixin::MdIdToUri do
+  before :each do
+    @client = ConnectionHelper.create_default_connection
+    project = ProjectHelper.get_default_project(client: @client)
+
+    class SomeTestClass
```
Comment (yenkins) [inline spec/integration/mixins/id_to_uri_spec.rb]: Has no descriptive comment (IrresponsibleModule)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/331/ : SUCCESS in 17s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/62/ : SUCCESS in 1m 38s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/372/ : SUCCESS in 27s

**Comment by panjan** [comment]
ok to merge

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/127/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/63/ : SUCCESS in 26s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/373/ : SUCCESS in 27s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/977/ : FAILURE in 8s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/738/ : ABORTED in 7m 14s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/128/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/64/ : SUCCESS in 28s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/374/ : SUCCESS in 1m 48s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/980/ : SUCCESS in 45m 15s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/739/ : SUCCESS in 24m 21s

**Pull request merged**